.travis.yml

```diff
@@ -1,10 +1,6 @@
 language: node_js
 sudo: false
 node_js:
-  - 4
-  - 5
-  - 6
```

Comment:
This repo is still being used by apps that use Node 6, so we shouldn't yet remove it here.

Reply:
Since this is going to be a major release. So I was thinking about dropping node 6 support.

Reply:
Ok, as long as this is a major release.